### ISSUE TITLE
cant get it working

### ISSUE THREAD

**Issue by Dbboli01** [carsakiller/Carsas-CommandsV2#71]
i follow the directions on the workshop and it de admins me in game on my server and all i get from the web server is  game script responded with an error: Timeout: Game not responding
will attach log
[log.txt](https://github.com/carsakiller/Carsas-CommandsV2/files/8477614/log.txt)

**Comment by carsakiller** [comment]
Hello @Dbboli01 ! 👋

Can you provide more info on the de-admin problem?
- What were you doing when it occurred?
- Can you confirm what your roles are with [`?playerRoles`](https://c2.carsakiller.com/cc-website/docs/commands#playerRoles)
- Can you confirm what your permissions are with [`?playerPerms`](https://c2.carsakiller.com/cc-website/docs/commands#playerPerms)
- Is this a dedicated server, a multiplayer game started from the main menu, or a singleplayer game?

As for the companion issues:
- Have you made sure that the companion is enabled in game with [`?setPref companion true`](https://c2.carsakiller.com/cc-website/docs/commands#setPref)?
- Were there any errors reporting in the chat?
- Are [these the instructions](https://c2.carsakiller.com/Carsas-Companion/manual.html#quickstart-guide) you followed for setting up the companion?

Thanks for attaching the logs, it helps a lot!

**Comment by Dbboli01** [comment]
?setPref companion true
when i tryed that it showed you have no permissions to do this
its dedicated server
no  errors in chat 
and yes used the install intructions as well as the ones on the git hub
any admins i had set where auto de admin
id have to work on it later i am currently busy so i cant check ?playerroles or ?playerPerms 

any admins that join while its running whould be deauthed and deadmined 
then re authed

**Comment by CrazyFluffyPony** [comment]
Make sure you give yourself Admin rights inside `%appdata%/Stormworks/server-config.xml`

```xml
	<admins>
		<id value="76561198054042332"/>
	</admins>
```
get your steamid here: https://steamid.io/

Then join the game, and write `?setPref companion true` into the chat


If that does not work, please also send us the log file of the companion server. It can be found in the same folder the companion server .exe is located.

**Comment by carsakiller** [comment]
Without knowing the results of `?playerRoles`/`?playerPerms`, it still sounds to me like you are not an owner.

The very first unique player to join the save is made an owner (this means the **very** first player, not the first player every time the server starts). This works very nicely for servers started from the main menu, but may not be ideal for dedicated servers.

To override this with v2, we added the ability for server owners to manually specify their steamID, disabling the auto-assigning of an owner. I have updated the [README](https://github.com/carsakiller/Carsas-CommandsV2#setting-an-owner) with detailed instructions on how this can be done.
